proto/cometbft/crypto/v1/keys.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package cometbft.crypto.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/crypto/v1";
+
+import "gogoproto/gogo.proto";
+
+// PublicKey is a ED25519 or a secp256k1 public key.
```

Comment:
This doc is not true as we have other keys supported (probably a change upstream for y'all)